src/styles/blocks/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
```

Comment:
The padding property is not consistent with the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). You should use shorthand properties whenever possible. Try to use the shorthand `padding` property here.